braket_features/Getting_Devices_and_Checking_Device_Properties.ipynb

```diff
@@ -0,0 +1,990 @@
+      "      \"fCZ_std_err\": 0.013536289599509059,\n",
+      "      \"fXY\": 0.9028915454624779,\n",
+      "      \"fXY_std_err\": 0.003875309777715083\n",
+      "    },\n",
+      "    \"5-6\": {\n",
+      "      \"fCZ\": 0.9748663918292196,\n",
+      "      \"fCZ_std_err\": 0.006644225111658855,\n",
+      "      \"fXY\": 0.9573385948832057,\n",
+      "      \"fXY_std_err\": 0.008834008846613742\n",
+      "    },\n",
+      "    \"6-7\": {\n",
+      "      \"fCZ\": 0.9597106644456523,\n",
+      "      \"fCZ_std_err\": 0.00915230083006339,\n",
+      "      \"fXY\": 0.959585777941374,\n",
+      "      \"fXY_std_err\": 0.006297039905522111\n",
+      "    }\n",
+      "  }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the IonQ device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/ionq/ionQdevice\")\n",
+    "\n",
+    "# the Rigetti device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/rigetti/Aspen-8\")\n",
+    "\n",
+    "execution_windows = device.properties.service.executionWindows\n",
+    "connectivity_graph = device.properties.paradigm.connectivity\n",
+    "calibration = device.properties.provider.specs\n",
+    "\n",
+    "print(f'The exeuation windows for {device.name}:\\n{execution_windows}\\n')\n",
+    "print(f'The connectivity graph of the qubits for this device:\\n {connectivity_graph}\\n')\n",
+    "print('Calibration data:\\n', json.dumps(calibration,sort_keys=True,indent=2))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The shots range of DW_2000Q_6 is (1, 10000)\n",
+      "The price of running tasks on this device: price=0.00019 unit='shot'\n",
+      "This device has 2048 qubits in total, with the topology: {'type': 'chimera', 'shape': [16, 16, 4]}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the DWave devices\n",
+    "# device = AwsDevice(\"arn:aws:braket:::device/qpu/d-wave/Advantage_system1\")\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6\")\n",
+    "\n",
+    "shots_range = device.properties.service.shotsRange\n",
+    "device_cost = device.properties.service.deviceCost\n",
+    "qubit_count = device.properties.provider.qubitCount\n",
+    "device_topology = device.properties.provider.topology\n",
+    "\n",
+    "print(f'The shots range of {device.name} is {shots_range}')\n",
+    "print('The price of running tasks on this device:', device_cost)\n",
+    "print(f'This device has {qubit_count} qubits in total, with the topology: {device_topology}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are many more device properties to explore and access for each device. Visit the [Amazon Braket schemas documentation](https://amazon-braket-schemas-python.readthedocs.io/en/latest/_apidoc/braket.device_schema.html) to learn more. "
```

Comment:
"Each device has more properties to explore. To learn more, view the [Amazon Braket schemas documentation](https://amazon-braket-schemas-python.readthedocs.io/en/latest/_apidoc/braket.device_schema.html)."